site-src/contributing/enhancement-requests.md

```diff
@@ -29,28 +29,21 @@ It is unlikely to require an enhancement if it:
 If you're unsure the proposed work requires an enhancement, file an issue
 and ask.
 
-## When to Create a New Enhancement
+## New Enhancement process
 
-Create an enhancement once you have:
+The new enhancement process is documented on the [GEP Overview][gep] page. Please
+see that page for all the details about how to log a new GEP, and the process
+it will follow on its journey towards Completed status.
 
-- Circulated your idea to see if there is interest.
-- Identified community members who agree to work on and maintain the enhancement.
-- Enhancements may take several releases to complete.
-- A prototype in your own fork (optional)
+A **documented** discussion of some form **must** exist prior to submitting a request for enhancement
+if that enhancement is non-trivial (which we will define as either: _implicates changes to the API specification_ OR _has some kind of end-user impact_). Please use our [Github Discussions][discussion]
+forum as the initial place to start, and feel free to bring that discussion up for synchronous conversation in
+one of our [community meetings][meetings]. If the created request doesn't include reference to a discussion and/or recordings of discussion in our community meetings, please note that it _may_ get closed
+with a request to create an initial discussion first.
```

Comment:
Nit: Wrapping at 80 chars would be nice for consistency with the rest of the file.

Reply:
Turns out "accept suggestion" in the Github UI is not an ideal way to do this. Fixed now.